R/new_covariate.R

```diff
@@ -70,21 +71,19 @@ new_covariate <- function(
       if(! (0 %in% new_times)) {
         # add zero time, and remove all t < 0
```

Comment:
the section below is just changing indentation. now the `else` logic is processed for all non-numeric information. i.e., LOCF is assumed for removing observations prior to t=0 for non numeric information. 

I thought about adding a check to ensure that `implementation == "LOCF"` for non-numeric data since interpolation does not make sense in that situation, but that's perhaps a slightly larger change to the interface. What do you think?

Reply:
I do think it's worth erroring or at least warning if there is a certain combination of arguments that isn't valid (and maybe putting that in the documentation too).